infer/src/pulse/PulseFormula.ml

```diff
@@ -2966,10 +3047,15 @@ module Formula = struct
 
     val normalize_atom : t -> Atom.t -> Atom.t list SatUnsat.t
 
-    val and_normalized_atoms : t * new_eqs -> Atom.t list -> (t * new_eqs) SatUnsat.t
+    val and_normalized_atoms :
+         t * new_eqs
+      -> Atom.t list
+      -> orig_atom:Atom.t list
+      -> add_term:bool
```

Comment:
maybe call this `add_raw_term` since this doesn't affect normal pulse formula operations?
```suggestion
      -> add_raw_term:bool
```